src/webgpu/web_platform/copyToTexture/canvas.spec.ts

```diff
@@ -97,7 +97,6 @@ class F extends CopyToTextureUtils {
     }
 
     const imageData = new ImageData(imagePixels, width, height, { colorSpace });
-    // MAINTENANCE_TODO: Remove as any when tsc support imageData.colorSpace
     /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
     if (typeof (imageData as any).colorSpace === 'undefined') {
```

Comment:
I usually don't see problems with TypeScript checking for undefined on things it thinks can't be undefined. I think because it's common for a JS library to need to dynamically typecheck incoming arguments if they're coming in from outside the library. Locally I'm unable to get either tsc or eslint to complain with this code:

```suggestion
    if (typeof imageData.colorSpace === 'undefined') {
```

But if a change is necessary, would this work?

```ts
    if (typeof (imageData.colorSpace as PredefinedColorSpace | undefined) === 'undefined') {
```